2018-03-28/azblob/zt_examples_test.go

```diff
@@ -316,7 +316,7 @@ func ExampleBlobSASSignatureValues() {
 	accountName, accountKey := accountInfo()
 
 	// Use your Storage account's name and key to create a credential object; this is required to sign a SAS.
-	credential := azblob.NewSharedKeyCredential(accountName, accountKey)
+	credential, _ := azblob.NewSharedKeyCredential(accountName, accountKey)
```

Comment:
given this is an example (and thus a recommended way of working with this library) - shouldn't this error be handled?